keps/sig-network/20190324-remove-kube-proxy-autocleanup.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Remove kube-proxy's automatic clean up logic
+authors:
+  - "@vllry"
+owning-sig: sig-network
+participating-sigs:
+reviewers:
+  - "@andrewsykim"
+approvers:
+  - "@bowei"
+  - "@thockin"
+editor: TBD
+creation-date: 2018-03-24
+last-updated: 2018-03-24
+status: provisional
+see-also:
+replaces:
+superseded-by:
+---
+
+# Remove kube-proxy's automatic clean up logic
+
+## Summary
+
+Remove the automatic network rule cleanup from kube-proxy.
+Only clean up rules from running in other proxy modes when using the `--cleanup` command.
+
+## Motivation
+
+kube-proxy's rule cleanup can substantially delay kube-proxy reboot time,
```

Comment:
Inline with the rest of the KEP, I think the motivation should include something about simplifying the clean up logic & making kube-proxy behavior more predictable (which should equal less bugs).